docs/tutorial-solver.md

```diff
@@ -1,49 +1,37 @@
-title: Quickstart - Running your own Solver
-sidebar_label: Quickstart
+title: Run a Solver
+sidebar_label: Overview
 ---
 
-## Contents
-
-1. [Installation](#installing-driver-tools)
-2. [Building The Driver](#building-the-driver)
-3. [Configuring Environment](#configuring-environment-variables)
-4. [Acquiring the Orderbook](#acquiring-the-orderbook)
-5. [Run the Solver](#run-the-solver)
-6. [Advanced Configuration](#advanced-configuration)
-   1. [Orderbook Filter](#Configuring-an-Orderbook-Filter)
-
----
+This tutorial will cover how to run the open solver for Gnosis Protocol. Before you get started, make sure you’ve familiarized yourself with the [Gnosis Protocol Introduction](https://docs.gnosis.io/protocol/docs/introduction1/), especially the section on what "solvers" are in the context of Gnosis Protocol.
 
 ## Installing Driver Tools
 
-Clone the dex-services repository, and acquire the project requirements
+Clone the dex-services repository, and acquire the project requirements:
 
 ```sh
 git clone https://github.com/gnosis/dex-services.git
 ```
 
-We demonstrate how to run the open solver.
-Once comfortable, the open solver can be adapted to suit your own needs from the [open solver repo](https://github.com/gnosis/dex-open-solver).
 
-## Building The Driver
+Once comfortable you're all set, the open solver can be adapted to suit your own needs from the [open solver repository](https://github.com/gnosis/dex-open-solver).
 
-Build the driver (with the open solver dockerhub image) from within the root directory of `dex-services` repo. This command builds the driver while simultanesouly baking the open solver into the the build.
+## Building the Driver
+
+Build the driver (with the open solver Docker Hub image) from within the root directory of `dex-services` repo. This command builds the driver while simultanesouly baking the open solver into the the build.
 
 ```sh
 docker-compose build --build-arg SOLVER_BASE=gnosispm/dex-open-solver:master stablex-debug
 ```
 
 Be aware, your Docker installation may require elevated privileges!
 
-Note that, if `SOLVER_BASE` is not specified, the driver's own internal `NaiveSolver` will be used.
-Alternatively, if you have your own personalized solver, this image would need to be specified during the build.
+Note that, if `SOLVER_BASE` is not specified, the driver's own internal `NaiveSolver` will be used. Alternatively, if you have your own personalized solver, this image would need to be specified during the build.
 
 ## Configuring Environment Variables
 
-The following three environment variables are _required_ when running the dex-services project.
-Note that the account corresponding to the private key should be funded with sufficient ETH to pay gas for solution submission.
+The following three environment variables are _required_ when running the dex-services project. Note that the account corresponding to the private key should be funded with sufficient ETH to pay gas for solution submission.
```

Comment:
Usually, in markdown, each sentence is on its own line. This is so that any changes made to one sentence are easily seen in the changelog 
```suggestion
The following three environment variables are _required_ when running the dex-services project. 
Note that the account corresponding to the private key should be funded with sufficient ETH to pay gas for solution submission.
```